tools/hangar/src/platform.test.ts

```diff
@@ -1,34 +1,35 @@
 import { describe, expect, test } from "vitest";
 import * as path from "path";
 import fs from "fs";
-import { pluginsDir, tmpDir } from "./paths";
+import { platformsDir, tmpDir } from "./paths";
 import {
   runWingCommand,
   sanitize_json_paths,
   tfResourcesOfCount,
 } from "./utils";
 import { Testing } from "cdktf";
 
-describe("Plugin examples", () => {
+describe("Platform examples", () => {
   const app = "main.w";
-  const appFile = path.join(pluginsDir, app);
+  const appFile = path.join(platformsDir, app);
 
-  describe("AWS target plugins", () => {
-    const args = ["compile", "--target", "tf-aws"];
-    const targetDir = path.join(pluginsDir, "target", "main.tfaws");
+  describe("AWS target platform", () => {
+    const args = ["compile"];
+    const platforms = ["tf-aws"]
+    const targetDir = path.join(platformsDir, "target", "main.tfaws");
 
     test("permission-boundary.js", async () => {
-      const plugin = path.join(pluginsDir, "permission-boundary.js");
+      const plugin = path.join(platformsDir, "permission-boundary.js");
+      platforms.push(plugin);
```

Comment:
nit (avoid mutating variables shared between tests)
```suggestion
    const basePlatforms = ["tf-aws"]
    const targetDir = path.join(platformsDir, "target", "main.tfaws");

    test("permission-boundary.js", async () => {
      const plugin = path.join(platformsDir, "permission-boundary.js");
      const platforms = [...basePlatforms, plugin];
```